### ISSUE TITLE
Humberger menu on mobile missing color

### ISSUE THREAD

**Issue by ChristianTonny** [Tech-Care-Rwanda/tech-care#70]
The humberger menu on mobile currently does not have a visible color and appears transparent. Add a background color or icon color for better accessibility and visibility on mobile devices.
